Documentation/cross-platform-performance-and-eventing.md

```diff
@@ -0,0 +1,287 @@
+# .NET Cross-Plat Performance and Eventing Design
+
+##Introduction
+
+As we bring up CoreCLR on the Linux and Mac platforms, it’s important that we determine how we’ll measure and analyze performance on these platforms.  On Windows we use an event based model that depends on ETW, and we have a good amount of tooling that builds on this approach.  Ideally, we can extend this model to Linux and Mac and re-use much of the Windows tooling.
+
```

Comment:
Given that a FreeBSD port has started (and others will no doubt follow), we should have an eye to ensuring that our choices are meaningful on other OSes, too. We certainly don't have to responsible for the implementation on every other OS, but we want the various port teams to be successful.

CC: @kangaroo, @janhenke